snap/hooks/remove

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
```

Comment:
did you run shellcheck against this?

We probably want to add it to our shellcheck target

Reply:
Yes and it's already checked by `make test-shellcheck`